content/en/docs/concepts/scheduling-eviction/pod-overhead.md

```diff
@@ -158,40 +156,43 @@ POD_ID="$(sudo crictl pods --name test-pod -q)"
 ```
 
 From this, you can determine the cgroup path for the Pod:
+
 ```bash
 # Run this on the node where the Pod is scheduled
 sudo crictl inspectp -o=json $POD_ID | grep cgroupsPath
 ```
 
 The resulting cgroup path includes the Pod's `pause` container. The Pod level cgroup is one directory above.
+
 ```
-        "cgroupsPath": "/kubepods/podd7f4b509-cf94-4951-9417-d1087c92a5b2/7ccf55aee35dd16aca4189c952d83487297f3cd760f1bbf09620e206e7d0c27a"
+  "cgroupsPath": "/kubepods/podd7f4b509-cf94-4951-9417-d1087c92a5b2/7ccf55aee35dd16aca4189c952d83487297f3cd760f1bbf09620e206e7d0c27a"
 ```
 
-In this specific case, the pod cgroup path is `kubepods/podd7f4b509-cf94-4951-9417-d1087c92a5b2`. Verify the Pod level cgroup setting for memory:
+In this specific case, the pod cgroup path is `kubepods/podd7f4b509-cf94-4951-9417-d1087c92a5b2`.
+Verify the Pod level cgroup setting for memory:
+
 ```bash
 # Run this on the node where the Pod is scheduled.
 # Also, change the name of the cgroup to match the cgroup allocated for your pod.
  cat /sys/fs/cgroup/memory/kubepods/podd7f4b509-cf94-4951-9417-d1087c92a5b2/memory.limit_in_bytes
 ```
 
 This is 320 MiB, as expected:
+
 ```
 335544320
 ```
 
 ### Observability
 
 A `kube_pod_overhead` metric is available in [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics)
-to help identify when PodOverhead is being utilized and to help observe stability of workloads
-running with a defined Overhead. This functionality is not available in the 1.9 release of
-kube-state-metrics, but is expected in a following release. Users will need to build kube-state-metrics
-from source in the meantime.
-
-
+to help identify when Pod overhead is being utilized and to help observe stability of workloads
+running with a defined overhead. This functionality is not available in the 1.9 release of
+`kube-state-metrics`, but is expected in a following release. Users will need to build
+`kube-state-metrics` from source in the meantime.
```

Comment:
> This functionality is not available in the 1.9 release of `kube-state-metrics`, but is expected in a following release. Users will need to build `kube-state-metrics` from source in the meantime.

We [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future); I'd drop this

Reply:
Double checked  the project and the metrics. The text needs some update.